common/src/KokkosFFT_transpose.hpp

```diff
@@ -290,12 +286,12 @@ void _transpose(const ExecutionSpace& exec_space, InViewType& in,
       });
 }
 
+// FIXME: not used?
```

Comment:
It is at least tested. The Dimension number is used for MDRange rank, but the maximum rank is 6. So I am performing a sequential operation inside parallel region.